src/GoodsList.tsx

```diff
@@ -5,12 +5,16 @@ type Props = {
   goods: Good[]
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
+export const GoodsList: React.FC<Props> = React.memo(({ goods }) => (
```

Comment:
React memo redundant here, because goods changed every time then you click on the button(getAll, get5...)